api_blog/views.py

```diff
@@ -1,11 +1,60 @@
+import json
+
 import sqlalchemy as sa
 from aiohttp import web
+from sqlalchemy.future import select
 
-from .models import Session
+from .models import Session, User
+from .security import generate_password_hash
 
 
 async def index(request):
     async with Session() as session:
         async with session.begin():
             await session.execute(sa.select(1))
     return web.Response(text="hello aiohttp")
+
+
+async def sign_up(request):
+    try:
+        data = await request.json()
+        user = data.get("username")
+        password = data.get("password")
+        response_obj = {
+            "status": "success",
+            "message": "new user created",
+            "username": user,
+        }
+
+        async with Session() as session:
+            async with session.begin():
+                session.add_all(
+                    [User(username=user, password=generate_password_hash(password))]
```

Comment:
If you know that only one user will be created why do you use add_all() ? It's always only one